package global.sesoc.java1.exception;

import java.util.Scanner;

public class ExTest2 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int n, s = 0;
		
		while (true) {
			System.out.print("정수 입력 : ");
			n = scan.nextInt();
			if (n == 0) break;
			s = s + n;
		}
		System.out.println("합계:" + s);
		
	}
}
